lib/package.json

```diff
@@ -36,8 +36,8 @@
     "dev": "npm run build watch"
   },
   "peerDependencies": {
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "react": ">=16 || >=17 || >= 18",
+    "react-dom": ">=16 || >=17 || >= 18"
```

Comment:
I wonder why we can't just use `>= 16` since it has the same effect as the sum of those ranges. None of them use the upper bound.
https://jubianchi.github.io/semver-check/#/%3E%3D16/19

The only difference I see on this semver-check is that if I use just `>= 16` it displays warning
> This constraint does not provide an upper bound which means you will probably get unexpected breaking changes.

Reply:
I changed it to `>=16 <=18`